pages/Armor-Forge.md

```diff
@@ -0,0 +1,14 @@
+The Armor Forge is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Armor Forge is a multiblock structure, it consists of a dispenser and an anvil.<br>
+**Note:** The dispenser has to be facing upwards and the anvil cannot be (slightly) broken.<br>
```

Comment:
"cannot be broken" sounds like you are not able to break the block.
I think you should rephrase it to:
"Note that it must be a standard anvil, damaged Anvils will not work".
And the fact about the dispenser is wrong, it doesn't have to be facing that way, it just looks nicer.

Reply:
I thought it had to be facing that way, since it says so in the sf guide

Reply:
But I'll definitely rephrase